xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.h

```diff
@@ -50,6 +50,8 @@ class CRenderManager;
 class IRenderMsg
 {
   friend CRenderManager;
+public:
+  virtual ~IRenderMsg() = default;
```

Comment:
> Do note that this is only relevant if you are (potentially) going to destroy the class via a pointer to the interface.

 ofc, but as you never know before and imo nobody checks the presence of a virtual dtor when first heap allocating a respective object and the price is just one more vtbl entry I tend to add this everywhere and be on the safe side.